omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -654,257 +770,361 @@
+    <span style="display: none" id="thirdLineRegimen">
+        <lookup
+                complexExpression="#foreach($third_line_regimen in $fn.allObs(162990)) $first_second_line_regimen.valueCoded #end"/>
+    </span>
 
-    <!--End of Lookup section-->
+    <span style="display: none" id="otherThirdLineRegimen">
+        <lookup
+                complexExpression="#foreach($other_third_line_regimen in $fn.allObs(162989)) $other_first_second_line_regimen.valueText #end"/>
+    </span>
 
-    <div>
-        <div class="row">
-            <div class="col-md-6">
-                <h1>HMIS HIV 003:HIV CARE/ART CARD - CLINICAL ASSESSMENT</h1>
-            </div>
-            <div class="col-md-5"></div>
-            <div class="col-md-1">
-                <span class="submitButton">
-                    <postSubmissionAction class="org.openmrs.module.aijar.htmlformentry.DSDSProgramSubmissionAction" />
-                    <submit class="confirm" submitLabel="Save"/>
-                </span>
+
+    <!-- This section contains lookup tags that pick data from other forms-->
+
+    <lookup complexExpression="#set($art_start_regimen='null') #set($other_art_start_regimen='null') #set($art_transfer_regimen='null') #set($other_art_transfer_regimen='null')"/>
+
+    <lookup complexExpression="#set($first_second_line_regimen='null') #set($other_first_second_line_regimen='null')"/>
+
+    <lookup complexExpression="#set($third_line_regimen='null') #set($other_third_line_regimen='null')"/>
+
+    <span style="display: none" id="artStartRegimen">
+        <lookup
+                complexExpression="#set($art_start_regimen=($fn.latestObs('99061').valueCoded))$art_start_regimen"/>
+    </span>
+
+    <span style="display: none" id="otherArtStartRegimen">
+        <lookup
+                complexExpression="#set($other_art_start_regimen=($fn.latestObs('99268').valueText))$other_art_start_regimen"/>
+    </span>
+
+    <span style="display: none" id="artTransferRegimen">
+        <lookup
+                complexExpression="#set($art_transfer_regimen=$fn.latestObs('99064').valueCoded)$art_transfer_regimen"/>
+    </span>
+
+    <span style="display: none" id="otherArtTransferRegimen">
+        <lookup
+                complexExpression="#set($other_art_transfer_regimen=$fn.latestObs('99269').valueText)$other_art_transfer_regimen"/>
+    </span>
+
+    <span style="display: none" id="firstSecondLineRegimen">
+        <lookup
+                complexExpression="#foreach($first_second_line_regimen in $fn.allObs(90315)) $first_second_line_regimen.valueCoded #end"/>
+    </span>
+
+    <span style="display: none" id="otherFirstSecondLineRegimen">
+        <lookup
+                complexExpression="#foreach($other_first_second_line_regimen in $fn.allObs(99146)) $other_first_second_line_regimen.valueText #end"/>
+    </span>
+
+
+    <span style="display: none" id="thirdLineRegimen">
```

Comment:
@slubwama remove all display:none and use class=hidden

Reply:
done